spec/rubocop/cop/style/each_with_object_spec.rb

```diff
@@ -24,6 +24,27 @@
     RUBY
   end
 
+  it 'finds inject and reduce with passed in and returned hash with single block argument' do
+    expect_offense(<<~RUBY)
+      [].inject({}) { |h| h[rand] = rand; h }
```

Comment:
When there is only one block argument, it’s better to use `expect_no_offenses` without registering an offense and raising an error. Because there is no point in using `each_with_object` through autocorrection by introducing the unused block argument `_`.

Reply:
Perhaps you're right, fixed. Thanks.